lib/ui/screens/collectible_found/collectible_found_screen.dart

```diff
@@ -82,10 +80,10 @@ class CollectibleFoundScreen extends StatelessWidget {
               Gap($styles.insets.lg),
               _buildRibbon(context),
               Gap($styles.insets.sm),
-              _buildTitle(context, collectible.title, $styles.text.h2, $styles.colors.offWhite, t * 1.5),
+              _buildTitle(context, collectible.title, $styles.text.h2, $styles.colors.offWhite, $styles.customTime.delay(450)),
               Gap($styles.insets.xs),
               _buildTitle(
-                  context, collectible.subtitle.toUpperCase(), $styles.text.title2, $styles.colors.accent1, t * 2),
+                  context, collectible.subtitle.toUpperCase(), $styles.text.title2, $styles.colors.accent1, $styles.times.med),
```

Comment:
I would just leave this whole method as-is, it should just work as it's all tied into `styles.times`?